docs/bootstrap/README.md

```diff
@@ -2,7 +2,12 @@
 
 1) [Sign up to DigitalOcean](https://www.digitalocean.com/?refcode=2962aa9e56a1&utm_campaign=Referral_Invite&utm_medium=Referral_Program&utm_source=CopyPaste)
 2) [Download Terraform](https://www.terraform.io)
-3) Clone this gist using the URL from the address bar
+3) Download the [cloud-config.tpl](cloud-config.tpl) and [main.tf](main.tf) files.
```

Comment:
I didn't have to do that when following the main blog post? https://www.openfaas.com/blog/faasd-tls-terraform/

Reply:
Right, because there is an instruction to clone the faasd repo earlier in the blog post, but not here. Should we also add that instruction here as well? 
I just thought it would make more sense if one only has to download the needed files `cloud-config.tpl` and `main.tf` here instead of cloning the whole faasd repo.
What would be better in your opinion?